.changeset/forty-timers-cheer.md

```diff
@@ -1,5 +1,6 @@
 ---
-'@backstage/plugin-scaffolder-backend': minor
+'@backstage/plugin-scaffolder-backend': major
```

Comment:
This can be still `minor` it's fine, we haven't made any breaking changes yet, just deprecations :)